Kakaotalk/Scenes/Splash/SplashViewModel.swift

```diff
@@ -9,18 +9,19 @@ import SwiftUI
 
 class SplashViewModel: ObservableObject {
     
-    @Published var shouldShowMainPage: Bool = false
+    @Published var shouldShowTabViewSection: Bool = false
     
     init() {
-    UITabBar.appearance().barTintColor = UIColor.white
+        UITabBar.appearance().barTintColor = UIColor.white
     }
     
-    func onViewApper() {
+    func onViewAppear() {
```

Comment:
The function name should be more descriptive. For example, `showTabView` or `navigateToMain`